@dolmasherpa123455
 
 > I'm considering adding the .npmrc file with engine-strict=true to enable strict enforcement of the engines field in the package.json file. If you don't mind, can I include that as part of this pull request?
 
 What would be the advantage of this parameter?
 
 Yarn Classic already treats the engine parameters as strict. This is the only package manager which needs to be considered.
 
 The project is not compatible with other package managers:
 npm install fails with "npm ERR! code ERESOLVE"
 pnpm run dev fails with "TSError: Unable to compile TypeScript:" 
 yarn run dev using Yarn Modern also fails with "TSError: Unable to compile TypeScript:"

--- a/package.json
+++ b/package.json
@@ -12,7 +12,7 @@
     "url": "https://github.com/cypress-io/cypress-realworld-app/issues"
   },
   "engines": {
-    "node": ">=12.22.0"
+    "node": "^16.16.0"
   },
   "dependencies": {
     "@auth0/auth0-react": "1.9.0",

